test/opsem/test_bo.ll

```diff
@@ -0,0 +1,225 @@
+; RUN: %seabmc "%s" 2>&1 | %oc %s
+; RUN: %seabmc --horn-bv2-lambdas --log=opsem3 "%s" 2>&1 | %oc %s
+
+; CHECK: ^sat$
+target datalayout = "e-m:o-p:32:32-f64:32:64-f80:128-n8:16:32-S128"
+target triple = "i386-apple-macosx10.14.0"
+
+; Function Attrs: nounwind ssp uwtable
+define i32 @main() #0 {
+  %1 = alloca i32, align 4
+  %2 = alloca i32, align 4
+  %3 = alloca i32, align 4
+  %4 = alloca i32, align 4
+  %5 = alloca i32, align 4
+  %6 = alloca i32, align 4
+  %7 = alloca i32, align 4
+  %8 = alloca i32, align 4
+  %9 = alloca i32, align 4
+  store i32 0, i32* %1, align 4
+  %10 = bitcast i32* %2 to i8*
+  call void @llvm.lifetime.start.p0i8(i64 4, i8* %10) #4
+  %11 = bitcast i32* %3 to i8*
+  call void @llvm.lifetime.start.p0i8(i64 4, i8* %11) #4
+  store i32 10, i32* %2, align 4, !tbaa !5
+  store i32 5, i32* %3, align 4, !tbaa !5
+  %12 = call i32 @nd()
+  %13 = icmp ne i32 %12, 0
+  br i1 %13, label %14, label %18
+
+; <label>:14:                                     ; preds = %0
+  %15 = load i32, i32* %2, align 4, !tbaa !5
+  %16 = load i32, i32* %3, align 4, !tbaa !5
+  %17 = sub nsw i32 %15, %16
+  store i32 %17, i32* %2, align 4, !tbaa !5
+  br label %18
+
+; <label>:18:                                     ; preds = %14, %0
+  %19 = load i32, i32* %2, align 4, !tbaa !5
+  %20 = icmp eq i32 %19, 5
+  br i1 %20, label %22, label %21
+
+; <label>:21:                                     ; preds = %18
+  call void @__VERIFIER_error() #5
```

Comment:
Something is wrong with this input. I don't expect any calls to `__VERIFIER_error`. One might think that this function indicates something to seahorn. Yet, under current set of options it is treated as `nd()`. It should either be removed, or this indicates some underlying issue with this test

Reply:
Ok, I will check the test and make changes accordingly.